src/components/events/EventCard/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Typography } from '@/components/common';
+import CommunityLogo from '@/components/common/CommunityLogo';
+import EventModal from '@/components/events/EventModal';
+import PointsDisplay from '@/components/events/PointsDisplay';
+import { PublicEvent } from '@/lib/types/apiResponses';
+import { formatEventDate } from '@/lib/utils';
+import Image from 'next/image';
+import { useState } from 'react';
+import styles from './style.module.scss';
+
+interface EventCardProps {
+  event: PublicEvent;
+  attended: boolean;
+}
+
+const EventCard = ({ event, attended }: EventCardProps) => {
+  const { cover, title, start, end, location } = event;
+  const [expanded, setExpanded] = useState(false);
+
+  let displayCover = cover;
+  if (!displayCover) {
+    displayCover = '/assets/graphics/store/hero-photo.jpg';
+  }
```

Comment:
this could be simplified to
```suggestion
  let displayCover = cover || '/assets/graphics/store/hero-photo.jpg';
```